README.md

```diff
@@ -1,2 +1,99 @@
-# website-template
-A template to create all public facing sites
+# Tiny Site Backend Setup Guide
+
+This README provides step-by-step instructions for setting up the Tiny Site Backend on your local development environment. The Tiny Site Backend is built using the Go programming language, PostgreSQL for the database, and Google authentication for user management. Before you begin, ensure that you have the following prerequisites installed on your system:
+
+1. Go 1.21
+2. PostgreSQL
+3. pgAdmin (optional but recommended for database management)
+
+## Prerequisites
+
+### Go 1.21
+
+Make sure you have Go version 1.21 or higher installed on your system. You can download and install Go from the official website: [Go Downloads](https://golang.org/dl/)
+
+### PostgreSQL
+
+You will need PostgreSQL as the database for the Tiny Site Backend. You can download and install PostgreSQL from the official website: [PostgreSQL Downloads](https://www.postgresql.org/download/)
```

Comment:
When I download PostgreSQl then it also downloaded pgAdmin

Reply:
No, You have to download it separately.

Reply:
The PostgreSQL installer includes pgAdmin for Windows and Mac, not sure about Linux.

![Screenshot_3](https://github.com/Real-Dev-Squad/tiny-site-backend/assets/87164140/952043e2-de9c-4379-bd06-5e7483462be1)

Reply:
Good